live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/repository/util/EventUtils.java

```diff
@@ -66,6 +71,18 @@ public final class EventUtils
     public static final String PREDICTION_REVIEW_STATUS_PROPERTY = "hxi:reviewStatus";
     public static final String PREDICTION_UNREVIEWED = "UNREVIEWED";
     public static final String PREDICTION_CONFIRMED = "CONFIRMED";
+    private static String digestAlgorithm;
+
+    @Value("${digestIdentifier.algorithm}")
+    public void setDigestIdentifierAlgorithm(String digestAlgorithm)
```

Comment:
This won't work because there won't be an instance of this class. Instead let's make the `digestAlgorithm` a parameter of the new method. We'll solve wiring up the Spring configuration in a later PR.